src/Microsoft.Health.Dicom.Operations/Microsoft.Health.Dicom.Operations.csproj

```diff
@@ -11,7 +11,7 @@
   <ItemGroup>
     <PackageReference Include="Ensure.That" Version="10.1.0" />
     <PackageReference Include="Microsoft.Azure.Functions.Extensions" Version="1.1.0" />
-    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.30" />
+    <PackageReference Include="Microsoft.Azure.WebJobs" Version="3.0.31" />
```

Comment:
Hopefully it won't cause the same problem as before again.

Reply:
I actually tested this by forcing the package import and it worked: https://microsofthealth.visualstudio.com/Health/_packaging?_a=package&feed=Internalfeed&package=Microsoft.Azure.WebJobs&protocolType=NuGet&version=3.0.31&view=overview

Reply:
Sweet!